secure/key-manager/src/lib.rs

```diff
@@ -253,9 +259,15 @@ where
 
     pub fn perform_action(&mut self, action: Action) -> Result<(), Error> {
         match action {
-            Action::FullKeyRotation => self.rotate_consensus_key().map(|_| ()),
-            Action::SubmitKeyRotationTransaction => self.resubmit_consensus_key_transaction(),
-            Action::NoAction => Ok(()),
+            Action::FullKeyRotation => {
+                COUNTERS.full_consensus_rotations.inc();
+                self.rotate_consensus_key().map(|_| ())
+            }
+            Action::SubmitKeyRotationTransaction => {
+                COUNTERS.rotation_transaction_resubmissions.inc();
```

Comment:
maybe better to put counters into the function handlers?

Reply:
The issue with this is that both rotation_consensus_key() and resubmit_transaction() call submit_transaction(). However, we're not counting the total number of transaction submissions, but only the number of resubmissions and rotations, so putting these statements into the function handlers will mess up the number of resubmission metrics...  

We could address this if we wanted to count the total number of submissions rather than resubmissions (see my comment above).